homeassistant/components/device_tracker/owntracks.py

```diff
@@ -31,25 +41,24 @@ def owntracks_location_update(topic, payload, qos):
             data = json.loads(payload)
         except ValueError:
             # If invalid JSON
-            logging.getLogger(__name__).error(
+            _LOGGER.error(
                 'Unable to parse payload as JSON: %s', payload)
             return
 
         if not isinstance(data, dict) or data.get('_type') != 'location':
             return
 
-        parts = topic.split('/')
-        kwargs = {
-            'dev_id': '{}_{}'.format(parts[1], parts[2]),
-            'host_name': parts[1],
-            'gps': (data['lat'], data['lon']),
-        }
-        if 'acc' in data:
-            kwargs['gps_accuracy'] = data['acc']
-        if 'batt' in data:
-            kwargs['battery'] = data['batt']
+        dev_id, kwargs = _parse_see_args(topic, data)
 
-        see(**kwargs)
+        # Block updates if we're in a region
+        with LOCK:
+            region = _in_region(dev_id)
```

Comment:
This method is only used once and basically boils down to this:

```python
if REGIONS_ENTERED[dev_id]:
```

I think we can remove that method.